pkg/service/server.go

```diff
@@ -77,20 +82,33 @@ func (mgr *Manager) Process(event *LifecycleEvent) error {
 }
 
 func (mgr *Manager) AddEvent(event *LifecycleEvent) {
-	mgr.workQueueSync.Lock()
+	var (
+		metrics   = mgr.metrics
+		queueSync = mgr.workQueueSync
+	)
+	queueSync.Lock()
 	event.SetEventTimeStarted(time.Now())
+	metrics.IncGauge("terminating_instances_count")
```

Comment:
add alb name as label/tag?

Reply:
Good idea, created https://github.com/keikoproj/lifecycle-manager/issues/32 for doing this